server/api/api.go

```diff
@@ -1134,8 +1147,11 @@ func jsonBytesResponse(w http.ResponseWriter, code int, json []byte) {
 	w.Write(json)
 }
 
-func errorResponse(w http.ResponseWriter, code int, message string, sourceError error) {
-	log.Printf("API ERROR %d, err: %v\n", code, sourceError)
+func errorResponse(a *API, w http.ResponseWriter, code int, message string, sourceError error) {
```

Comment:
If we are going to pass the whole `API` object here, always, I think it should be a method. if not I would prefere to pass only the `a.logger` that is the only needed part from the API.

Reply:
Done